authentication/models.py

```diff
@@ -21,7 +21,7 @@ def create_superuser(self, email, password=None, **extra_fields):
 
 
 class CustomUser(AbstractBaseUser, PermissionsMixin):
-    email = models.EmailField(max_length=50, unique=True)
+    email = models.EmailField(unique=True)
```

Comment:
Is this correct? Isn't `max_length` mandatory here?

Reply:
The email field has a default max_length of 254 characters.